contracts/contracts/harvest/OETHBaseHarvester.sol

```diff
@@ -0,0 +1,234 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import { IVault } from "../interfaces/IVault.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { ISwapRouter } from "../interfaces/aerodrome/ISwapRouter.sol";
+
+contract OETHBaseHarvester is Governable {
+    using SafeERC20 for IERC20;
+
+    event YieldSent(address recipient, uint256 yield, uint256 fee);
```

Comment:
Events should be together, after storage and before modifiers.

Reply:
Fixed